### PR TITLE
Change TTL

### PR DESCRIPTION
**Pull request by Cuspid88**
Depends on
[https://github.com/martenframework/marten/pull/249](https://github.com/martenframework/marten/pull/249)

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -10,7 +10,9 @@ crystal: ">= 1.6.0"
 
 dependencies:
   marten:
-    github: martenframework/marten
+    #github: martenframework/marten
+    github: Cuspid88/marten
+    branch: session_expires
```
Comment (ellmetha) [inline shard.yml]: Can we revert this? 🙏 
The `shard.yml` file shouldn't be impacted by this change.

File: src/marten_redis_session/store.cr [hunk 2]
```diff
@@ -60,9 +60,7 @@ module MartenRedisSession
 
     private def persist_session_data(data = nil)
       data = data.nil? ? "{}" : data.to_json
-      expires_in = Time::Span.new(seconds: Marten.settings.sessions.cookie_max_age)
-
-      client.set(client_key(@session_key.not_nil!), data, expires_in.total_seconds.to_i)
+      client.set(client_key(@session_key.not_nil!), data, expires_in)
```
Comment (ellmetha) [inline src/marten_redis_session/store.cr:63]: Can we add a unit test verifying the expiry is indeed set appropriately? 🙏

File: shard.yml (hunk 1 above)
Comment (Cuspid88) [reply shard.yml]: Sorry. It's a bug. I'll fix it now.

File: src/marten_redis_session/store.cr (hunk 2 above)
Comment (Cuspid88) [reply src/marten_redis_session/store.cr:63]: Of course. I'll do everything.

**Comment by Cuspid88** [comment]
Note: tested with the main branch.

shard.lock
```yml
  marten:
    git: https://github.com/martenframework/marten.git
    version: 0.5.1+git.commit.c1afb28410aefab33bf1ff985de2284f82146bbe
```

**Review by ellmetha — APPROVED** [review]
Looks good. Thanks for adding this! 🙏

**Pull request merged**